processing/src/main/java/org/apache/druid/frame/write/InvalidNullByteException.java

```diff
@@ -53,17 +50,16 @@ private InvalidNullByteException(
       @Nullable final Integer position
   )
   {
-    super(StringUtils.format(
+    super(column, StringUtils.format(
```

Comment:
Feels weird that we have this extra layer of indirection, to what will essentially call the `RuntimeException`. If we don't extract the relevant fields (`.getColumn()`) from the `FrameFieldWriterException` anywhere, or use check for it's instanceof, then I suppose we can get rid of it.

Reply:
+1

Reply:
Converting to MSQFault now for which the specific exception class is required. `InvalidNullByteException` now extends `RuntimeException` directly.